frontend/src/views/Org/GroupPage/components/GroupMembersSection/GroupMembersTable.tsx

```diff
@@ -0,0 +1,195 @@
+import { useMemo } from "react";
```

Comment:
I get this error when attempting to invite an incomplete user to the group 

<img width="1512" alt="image" src="https://github.com/user-attachments/assets/c1956d0e-d2c9-4b9d-adca-b06f562488c6">

Reply:
cant reproduce